site/en/tutorials/curl_quickstart.ipynb

```diff
@@ -0,0 +1,355 @@
+{
```

Comment:
<p>It looks like <code>stopSequence</code> is actually <code>stopSequences</code> (plural) and takes a list.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
done